recipes/cc65/all/conanfile.py

```diff
@@ -0,0 +1,175 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, MSBuild, tools
+from conans.client.tools.win import msvs_toolset
+from conans.errors import ConanInvalidConfiguration
+from contextlib import contextmanager
+import os
+
+
+class Cc65Conan(ConanFile):
+    name = "cc65"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://cc65.github.io/"
+    description = "A freeware C compiler for 6502 based systems"
+    license = "zlib"
+    topics = ("conan", "cc65", "compiler", "cmos", "6502", "8bit")
+    exports_sources = "patches/**"
+
+    settings = "os_build", "arch_build", "compiler"
+
+    _autotools = None
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.compiler == "Visual Studio":
+            if self.settings.arch_build not in ("x86", "x86_64"):
+                raise ConanInvalidConfiguration("Invalid arch_build")
+
+    def build_requirements(self):
+        if tools.os_info.is_windows:
+            # msys2 provides make for MSVC and mingw + install for mingw
+            self.build_requires("msys2/20190524")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
```

Comment:
if possible, add some link where patch was taken from (issue, commit, etc.), if it's not something written by you

Reply:
Patches were written by me.

Reply:
Actually, it's one small patch that will make the behavior of cc65 the same for Linux and Windows.